### DISCUSSION TITLE
[Switch] add readonly prop ?

### DISCUSSION THREAD

**Discussion by vaynevayne** [mantinedev/mantine#3833] [Feature requests]
<img width="1216" alt="image" src="https://user-images.githubusercontent.com/124666577/226283972-5e5f5a6a-b8c0-4fa9-97f5-9789e72b380d.png">

<img width="1201" alt="image" src="https://user-images.githubusercontent.com/124666577/226283928-25717c0b-52b2-456f-9b49-6d98c334b109.png">

**Comment by rtivital** [comment]
Switch component supports all props from `input[type="checkbox"]`, you can use any of the native input compatible props.

**Reply by vaynevayne** [reply]
thanks
